BYT/View Controllers/FoaasPreviewViewController.swift

```diff
@@ -47,8 +48,19 @@ class FoaasPrevewViewController: UIViewController, FoaasTextFieldDelegate {
   }
   
   
-  // MARK: - Actions
+  // MARK: - FoaasButtonDelegateMethods
   
+    internal func backButtonPressed() {
+        _ = self.navigationController?.popViewController(animated: true)
+    }
+    
+    internal func doneButtonPressed() {
+        let messageAndSubtitle = self.foaasPreviewView.previewTextView.text.components(separatedBy: "\n")
+        let notificationCenter = NotificationCenter.default
+        notificationCenter.post(name: Notification.Name(rawValue: "FoaasObjectDidUpdate"), object: Foaas(message: messageAndSubtitle[0], subtitle: messageAndSubtitle[1]))
```

Comment:
passing back the string like this makes sense to me. @anama118118, can you work with this or do you need the value passed to be something different?